pipe_tools/timestamp.py

```diff
@@ -139,24 +137,6 @@ def rfc3339strFromTimestamp(ts):
 def timestampFromRfc3339str(s):
     return timestampFromUdatetime(udatetime.from_string(s))
 
-def str2date(s):
```

Comment:
Why did you remove these method? How are you sure it isn't being used in any dataflow process?

Reply:
@enriquetuya These methods I added to work with BigQueryOperator. They were moved to the `airflow-gfw` repo. The link to the file that it contains the methods is [this](https://github.com/GlobalFishingWatch/airflow-gfw/pull/1/files#diff-f40d9a34dfaaa298eec29db28500f2e6)